2022-10-10-report.org

```diff
@@ -0,0 +1,95 @@
+- Fix -Wunused using `rust-lang.cc` hack [[https://github.com/rust-gcc/gccrs/pull/1560][PR1560]]
+- Fix include structure [[https://github.com/rust-gcc/gccrs/pull/1559][PR1559]]
+- testing: try loop in const function [[https://github.com/rust-gcc/gccrs/pull/1554][PR1554]]
+- Dump item visibility [[https://github.com/rust-gcc/gccrs/pull/1547][PR1547]]
+
+*** Contributors this week
+
+- [[https://github.com/ibuclaw][Iain Buclaw]]
+- [[https://github.com/dafaust][David Faust]]
+- [[https://github.com/abbasfaisal][Faisal Abas]]
+
+*** Overall Task Status
+
+| Category    | Last Week | This Week | Delta |
+|-------------+-----------+-----------+-------|
+| TODO        |       169 |       175 |    +6 |
+| In Progress |        31 |        31 |     - |
+| Completed   |       460 |       467 |    +7 |
+
+*** Test Cases
+
+| TestCases | Last Week | This Week | Delta |
+|-----------+-----------+-----------+-------|
+| Passing   | 6783      | 6803      | +20   |
+| Failed    | -         | -         | -     |
+| XFAIL     | 52        | 52        | -     |
+| XPASS     | -         | -         | -     |
+
+*** Bugs
+
+| Category    | Last Week | This Week | Delta |
+|-------------+-----------+-----------+-------|
+| TODO        |        52 |        53 |    +1 |
+| In Progress |        17 |        14 |    -3 |
+| Completed   |       204 |       211 |    +7 |
+
+*** Milestones Progress
+
+__FIXME__: Fix milestones! Add issues to the milestones!
+
+- Close out the const generic one
+- Keep the intrinsics one? Or is that a project more than a milestone?
+- Identify which intrinsics are for libcore and which are for after
+
+| Milestone                         | Last Week | This Week | Delta | Start Date     | Completion Date | Target        |
+|-----------------------------------+-----------+-----------+-------+----------------+-----------------+---------------|
+| Data Structures 1 - Core          |      100% |      100% | -     | 30th Nov 2020  | 27th Jan 2021   | 29th Jan 2021 |
+| Control Flow 1 - Core             |      100% |      100% | -     | 28th Jan 2021  | 10th Feb 2021   | 26th Feb 2021 |
+| Data Structures 2 - Generics      |      100% |      100% | -     | 11th Feb 2021  | 14th May 2021   | 28th May 2021 |
+| Data Structures 3 - Traits        |      100% |      100% | -     | 20th May 2021  | 17th Sept 2021  | 27th Aug 2021 |
+| Control Flow 2 - Pattern Matching |      100% |      100% | -     | 20th Sept 2021 | 9th Dec 2021    | 29th Nov 2021 |
+| Macros and cfg expansion          |      100% |      100% | -     | 1st Dec 2021   | 31st Mar 2022   | 28th Mar 2022 |
+| Imports and Visibility            |      100% |      100% | -     | 29th Mar 2022  | 13th Jul 2022   | 27th May 2022 |
+| Const Generics                    |       76% |       76% | -     | 30th May 2022  | -               | 17th Oct 2022 |
+| Intrinsics and builtins           |       15% |       15% | -     | 6th Sept 2022  | -               | 14th Nov 2022 |
```

Comment:
Do you want to add in the other milestones we created?

Reply:
Yes, they were still missing some issues up until this morning but that's now done :) I added them in 242f971